schema/elasticsearch/v7/visibility/versioned/v1/reindex.json

```diff
@@ -0,0 +1,17 @@
+{
+  "conflicts": "proceed",
+  "source": {
+    "index": "temporal-visibility-dev"
```

Comment:
why the -dev suffix?

Reply:
I guess, it is legacy. But it is a good point to remove that dash from the schema template and also use parameter for both old and new schema names.